DragaliaAPI.Database/Entities/DbPlayerCharaData.cs

```diff
@@ -34,8 +34,43 @@ public class DbPlayerCharaData : IDbHasAccountId
     [Column("Level")]
     public byte Level { get; set; } = 1;
 
-    [Column("AddMaxLevel")]
-    public byte AdditionalMaxLevel { get; set; } = 0;
+    [NotMapped]
+    public byte AdditionalMaxLevel
```

Comment:
could you add a short comment explaining how this works, I think you told me but I already forgot :upside_down_face: